src/components/MovieCard/MovieCard.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { MovieCard } from './index';
+import { Meta, StoryObj } from '@storybook/react';
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
I suggest to disable this rule on `.eslintrc`

Reply:
this is ok!